docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  development:
+    image: todo-app:development
+    env_file:
+      - .env
+    ports:
+      - "5000:5000"
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /opt/todoapp/todo_app
+
+  production:
+    image: todo-app:production
+    env_file:
+      - .env
+    ports:
+      - "5000:80"
```

Comment:
I'd be tempted to run this on another port so that we can run both this and dev at the same time, but definitely optional